stac_mjx/utils.py

```
@@ -0,0 +1,313 @@
+"""This module contains utility functions for Stac."""
+
+import os
+
+import jax
+from jax import numpy as jp
+from jax.lib import xla_bridge
+from mujoco import mjx
+from mujoco.mjx._src import smooth
+import numpy as np
```

Ruff (0.8.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)